SIS_import/sis_set_url.rb

```diff
@@ -523,14 +525,18 @@ def read_argv
 
 			esbToken=refreshESBToken()
 
-			# update MPathway with Canvas urls
-			updateError = update_MPathway_with_Canvas_url(esbToken, outputDirectory)
+			if (!esbToken.nil?)
+				# update MPathway with Canvas urls
+				updateError = update_MPathway_with_Canvas_url(esbToken, outputDirectory)
```

Comment:
Maybe call this possibleUpdateError? Or updateErrorMsg?  Otherwise one might wonder why the method will always return an updateError.